generator-cas-app/README.md

```diff
@@ -0,0 +1,18 @@
+# generator-cas-app
+> Generates various scaffolds for a new CAS application.
+
+## Installation
+
+```bash
+npm install -g yo
+npm link
```

Comment:
use yarn instead?

Reply:
The thing with yarn at this point, is that asdf hasn't been run yet, so yarn will have to be installed another way.

Reply:
But it's also the same thing with node as well... 🤷🏻